app/app.go

```diff
@@ -967,6 +975,15 @@ func New(
 	app.HardForkManager = upgrades.NewHardForkManager(app.ChainID)
 	app.HardForkManager.RegisterHandler(v0upgrade.NewHardForkUpgradeHandler(100_000, upgrades.ChainIDSeiHardForkTest, app.WasmKeeper))
 
+	if app.evmRPCConfig.LiveEVMTracer != "" {
```

Comment:
let's put this config under either https://github.com/sei-protocol/sei-chain/blob/main/x/evm/querier/config.go or a new config section, since it's used in the actual processing and not an RPC-side thing

Reply:
What about:

```
...

[evm_tracing]
live_tracer = <name>

...
```

Does that sounds good?